app/scripts/models/auth_brokers/fx-desktop.js

```diff
@@ -183,6 +219,7 @@ define([
         'unwrapBKey',
         'keyFetchToken',
         'customizeSync',
+        'declinedSyncEngines',
```

Comment:
This really does seem backwards to me, how come declined instead of accepted again?